opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -554,6 +554,38 @@ impl Default for BatchConfig {
     }
 }
 
+impl BatchConfig {
```

Comment:
For those methods, it's worth giving more context. For example, what is `max_queue_size`, and how it can affect the batch span processor? A good reference will be common in `BatchConfig`'s fields.